saleor/product/migrations/0019_auto_20161208_0324.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2016-12-08 09:24
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('product', '0018_auto_20161207_0844'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
Can't we change the original migration instead? It's much easier to do it on a branch than to squash them later on master.